public/app/features/search/page/components/ManageActions.tsx

```diff
@@ -1,31 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import { Button, Checkbox, HorizontalGroup, useStyles2 } from '@grafana/ui';
+import { contextSrv } from 'app/core/services/context_srv';
+import { FolderDTO, FolderInfo } from 'app/types';
+
+import { GENERAL_FOLDER_UID } from '../../constants';
 
 import { getStyles } from './ActionRow';
+import { ConfirmDeleteModal } from './ConfirmDeleteModal';
+import { MoveToFolderModal } from './MoveToFolderModal';
 
 type Props = {
   items: Map<string, Set<string>>;
+  folder?: FolderDTO; // when we are loading in folder page
```

Comment:
rather than `onMoveItems`... should we pass in a callback that will get called when something changes?  I don't think we need the actual info, just an indication that the action happened and we should update the UI

Reply:
that's a good point, I'll do that in future PR when we get to how we'll update the item list :)